scss/_pagination.scss

```diff
@@ -149,15 +154,15 @@
       background-color: var(--#{$prefix}pagination-active-item-bg);
       border-color: var(--#{$prefix}pagination-active-item-border-color);
     }
+
+    &:not([title]) {
+      padding-inline: var(--#{$prefix}pagination-padding-inline-x);
```

Comment:
Might be a bit too soon for this https://caniuse.com/mdn-css_properties_padding-inline (but we use gap sometimes so idk). I think we should stick to `--bs-pagination-padding-x` here